src/test/java/specs/AroundSpecs.java

```diff
@@ -71,8 +71,7 @@
 
         assertThat(result.getFailureCount(), is(1));
         assertThat(result.getFailures().get(0), is(failure("first spec",
-            RuntimeException.class, "aroundEach did not run the block")));
-
+            RuntimeException.class, "At least one of the test hooks did not run the test block.")));
```

Comment:
With the generic hooks implementation, we certainly can't give as specific a message. Maybe we can at least include the location/trace to the offending hook?